pkg/virt-controller/watch/migration.go

```diff
@@ -1342,6 +1340,44 @@ func (c *MigrationController) sync(key string, migration *virtv1.VirtualMachineI
 	return nil
 }
 
+func (c *MigrationController) setupTargetPodCreation(vmi *virtv1.VirtualMachineInstance) error {
```

Comment:
The function name is too generic and does not reflect the real purpose. I would name it more explicitly, e.g.:

```suggestion
func (c *MigrationController) setupVMIRuntimeUser(vmi *virtv1.VirtualMachineInstance) error {
```